src/ResourceManager/Compute/Commands.Compute.ManagedService/Microsoft.Azure.Commands.Compute.ManagedService.format.ps1xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If the format.ps1xml files have no contents, then remove them and remove the references.